app/ui/src/assets/dictionary/en-GB.json

```diff
@@ -20,6 +20,10 @@
       "customizations": "Customizations",
       "settings": "Settings"
     },
+    "links": {
+      "userguide": "https://access.redhat.com/documentation/en-us/red_hat_fuse/7.0/html-single/integrating_applications_with_ignite/{{0}}",
+      "tutorial": "https://access.redhat.com/documentation/en-us/red_hat_fuse/7.0/html-single/ignite_sample_integration_tutorials/"
```

Comment:
Just a question. I see that the userguide link has a placeholder at the end of it. I assume that's for jumping to a particular location in the user guide. And it must be okay if this gets replaced with null since the Help > User Guide link jumps to the top of the document. But the tutorial link does not have a placeholder at the end of it. If we ever want to link to a particular tutorial, and not just the "book", would that {{0}} placeholder need to be added? Should it just be added now?